src/app/ckeditor/ckeditor-playground/ckeditor-playground.component.ts

```diff
@@ -0,0 +1,182 @@
+import { Component, Inject, OnInit, ViewChild } from '@angular/core';
+import { TextValueAsXMLComponent } from '@dasch-swiss/dsp-ui/lib/viewer/values/text-value/text-value-as-xml/text-value-as-xml.component';
+import {
+    KnoraApiConnection,
```

Comment:
organize imports

Reply:
done in https://github.com/dasch-swiss/dsp-ui-lib/pull/134/commits/2e7b3456557533e626891e138655faa23a444808